packages/components/src/highlight-cards/highlight-cards-heading.tsx

```diff
@@ -0,0 +1,13 @@
+import { useTranslate } from 'i18n-calypso';
+
+export default function HighlightCardsHeading( { children }: { children: React.ReactNode } ) {
+	const translate = useTranslate();
+	return (
+		<div className="highlight-cards-heading__wrapper">
+			<h3 className="highlight-cards-heading">{ children }</h3>
+			<div className="highlight-cards-heaing__update-frequency">
```

Comment:
Is the class name missing a 'd' in heading? To be: `highlight-cards-heading__update-frequency`

Reply:
Lol wow! Its crazy what the eyes overlook until pointed out. I carried that over to two different style rules as well. 🤣  Updated.

Reply:
Same 😅 At least you were consistent, so it still worked. 😂